formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/rer/FormstackPerformRerHandler.scala

```diff
@@ -19,7 +19,7 @@ case class FormstackPerformRerHandler(
   def initiateRer(request: RerPerformRequest): Either[Throwable, S3WriteSuccess] =
     for {
       submissionIds <- dynamoClient.userSubmissions(request.subjectEmail.toLowerCase, config.bcryptSalt, config.submissionTableName)
-      _ <- formstackClient.deleteUserData(submissionIds, config)
+      _ <- formstackClient.deleteUserData(request.subjectEmail.toLowerCase, submissionIds, config)
```

Comment:
see comment below, but basically the the `email` field of the submissionsIds doesn't contain the email as I was assuming, it actually has a hash of the email, so I pass the actual email in the request to use that instead